source/GameData.cpp

```diff
@@ -1387,10 +1405,161 @@ void GameData::PrintWeaponTable()
 		cout << hull << '\t';
 		double hitforce = outfit.HitForce() * 60. / outfit.Reload();
 		cout << hitforce << '\t';
+        double heatdmg = outfit.HeatDamage() * 60. / outfit.Reload();
+        cout << heatdmg << '\t';
+        double slow = outfit.SlowingDamage() * 60. / outfit.Reload();
+        cout << slow << '\t';
+        double disruption = outfit.DisruptionDamage() * 60. / outfit.Reload();
+        cout << disruption << '\t';
+        double ion = outfit.IonDamage() * 60. / outfit.Reload();
+        cout << ion << '\t';
 		
 		cout << outfit.Homing() << '\t';
-		double strength = outfit.MissileStrength() + outfit.AntiMissile();
-		cout << strength << '\n';
+        cout << outfit.MissileStrength() << '\t';
+        cout << outfit.Velocity() << '\n';
+
 	}
 	cout.flush();
 }
+
+
+
+void GameData::PrintTurretTable()
```

Comment:
This ties in with the comment before this one, I can do that.

Reply:
I'm not sure the best way to parameterize it though, it would be possible to pass in a category string but that would mean AM are lumped in with Turrets. And, to be honest, I think most people think of them as a whole separate category of weapon, like Guns vs. Secondary Weapons even though they both go in gun ports they serve different roles, so having AM with Turrets would be annoying.
(This is obviously out of scope but I think Anti-Missile should be a category like Secondary Weapons.)